Presentation.Web/app/components/it-contract/it-contract-plan.controller.ts

```diff
@@ -874,6 +877,8 @@
                 ]
             };
 
+            Helpers.UiCustomizationHelper.removeUnavailableColumns(mainGridOptions.columns);
```

Comment:
was this not already called on this overview?

Reply:
as far as I remember you added it while doing the one with roles, so why call it again?... and the blue print should also be there right?

Something is off here or not up to date

Reply:
![image](https://user-images.githubusercontent.com/13116889/172606567-4b5df926-d7a9-4342-aec6-a9d0fc99ad92.png)

**And the blue print**

![image](https://user-images.githubusercontent.com/13116889/172606718-5ff63e6f-0bfe-4daa-a39b-7c7a646996b2.png)